src/legacy/core_plugins/kibana/public/dashboard/_dashboard_app.scss

```diff
@@ -1,7 +1,7 @@
 .dshAppContainer {
-  flex: 1;
   display: flex;
   flex-direction: column;
+  height: 100%;
```

Comment:
👍 FWIW, I see this same change was tested and approved in Joe's PR - https://github.com/elastic/kibana/pull/48913/files#diff-3edabc97f406f1537759321bb012ec14R4